rag_apps/hybrid_rag_system/app.py

```
@@ -0,0 +1,446 @@
+    from mistralai.client import Mistral
+    api_key = os.environ.get("MISTRAL_API_KEY", "")
+    if not api_key:
+        return None
+    return Mistral(api_key=api_key)
+
+
+@st.cache_resource(show_spinner=False)
+def _get_collection():
+    try:
+        return get_chroma_collection()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)